README.md

```diff
@@ -1 +1,68 @@
+>
+>**Conclusion** The incident is likely to have had a measurable impact on cases and infections in the affected areas in the South West of England.
+
+### **Serious Untoward Incident** 
+The Serious Untoward Incident Investigation report can be found by accessing the following [link](https://www.gov.uk/government/publications/serious-untoward-incident-investigation-immensa-health-clinic-limited)
+
+## Repository Organization
+
+
+    │── data               <- Holds data files
+    │
+    ├── outputs
+    │ └── report           <- Where the Rmarkdown preprint will be outputted       
+    │ └── visuals          <- Plots and visualisations will be created within this folder
+    │
+    │── main_script.R      <- Runs the analysis and produces the preprint
+    │
+    ├── src                <- Source code for use in this project
+    │ └── example_foo.R    <- Example R file containing functions used in scripts.
+    │
+    ├── .Rproj             <- R project for analysis
+    │
+    ├── SECURITY.md        <- Security terms and conditions
+    │
+    └── README.md          <- The top-level README for developers using this project
+
+
+
+Replication
+----------
+This repository contains the code & data for the preprint publication (INSERT LINK HERE). The code is written in R, and data stored in csv format and a shape file will be called from the ONS api and stored as a geojson after the first run.
+
+- All plots and tables from the paper can be reproduced using the functions in the `src` folder.
+
+- Data, from a variety of sources described in the publication, can be found in the `data` folder. A description of datasets is in the readme in the folder.
+ 
+- Outputs, including visualisations and the publication, can be found in the `outputs` folder.
+
+A master script `main_script.R` can be found in the main folder. To replicate the analysis and preprint run the script.
+
+This will run all of the scripts for each separate analysis discussed in the paper (e.g. descriptive plots, causal impact ... ).
+
+Extensions
+----------
+Data and code from this project have been published in the spririt of collaboration. Please feel free to replicate this work, or to adapt & develop the work further. When using the data and code from this repository please cite using the following.
+
+> Hounsome, L., et al. (2022). Epidemiological impact of a large number of incorrect negative SARS-CoV-2 test results in South West England during September and October 2021. https://doi.org/10.1101/2022.11.30.22282922 

```

Comment:
Suggested alternative: "Hounsome, L., Herr, D., Bryant, R., Smith, R., Loman, L., Harris, J., Youhan, U., Dzene, E., Hadjipantelis, P., Long, H. and Laurence, T., 2022. Epidemiological impact of a large number of incorrect negative SARS-CoV-2 test results in South West England during September and October 2021. medRxiv, pp.2022-11."

Reply:
Actioned